plugins/logs/plugin_test.go

```diff
@@ -1705,6 +1705,7 @@ func TestPluginMasking(t *testing.T) {
 					if tc.errManager.Error() != err.Error() {
 						t.Fatalf("expected error %s, but got %s", tc.errManager.Error(), err.Error())
 					}
+					return
```

Comment:
My understanding here is that this is an error in the tests and that after testing the error message, the test should complete in these cases.

Without this, the test continues and errors here:

```go
if err := plugin.maskEvent(ctx, nil, event); err != nil {
  t.Fatal(err)
}
```

Since we have subtly changed the behaviour of parseModules to always check the contents of the store, I think that this was just skipped when we had the old logic (`len(r.modules) == 0`) in place.

Please check my thinking here though!

Reply:
Good catch! As best I can tell, you're correct here.